core/src/execution/datafusion/shuffle_writer/shuffle_writer_test.rs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//! unit tests for the shuffle writer
+
+use super::*;
+
+#[test]
+fn test_slot_size() {
```

Comment:
Could you put this unit test in `shuffle_writer.rs` as usual?

Reply:
Usually the unit test code is in same file in Rust as I see. You can see we also do it for other Rust files.